CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 # Changelog
 
-## Unreleased
+# Unreleased
```

Comment:
Is this change intensional?

Reply:
good question, I copied from the crypto-org/ethermint changelog format but I think it might be better to keep it as before, reverted.